tests/behat/features/plugin-check-severity.feature

```diff
@@ -192,5 +192,33 @@ Feature: Test that the severity level in plugin check works.
       upgrade_notice_limit,WARNING,5
       """
 
+    When I run the WP-CLI command `plugin check foo-bar-wp --format=csv --fields=code,type,severity --error-severity=7 --include-low-severity-errors`
+    Then STDOUT should contain:
+      """
+      allow_unfiltered_uploads_detected,ERROR,7
+      """
+    And STDOUT should contain:
+      """
+      WordPress.WP.AlternativeFunctions.rand_mt_rand,ERRORS_LOW_SEVERITY,5
+      """
+    And STDOUT should contain:
+      """
+      WordPress.Security.EscapeOutput.OutputNotEscaped,ERRORS_LOW_SEVERITY,5
+      """
+
+    When I run the WP-CLI command `plugin check foo-bar-wp --format=csv --fields=code,type,severity --warning-severity=7 --include-low-severity-warnings`
+    Then STDOUT should contain:
+      """
+      allow_unfiltered_uploads_detected,ERROR,7
+      """
+    And STDOUT should contain:
+      """
+      upgrade_notice_limit,WARNINGS_LOW_SEVERITY,5
+      """
+    And STDOUT should contain:
+      """
+      default_readme_text,WARNING,7
```

Comment:
This error code `default_readme_text` has been converted to ERROR few weeks back.